src/astgen.zig

```diff
@@ -963,7 +963,7 @@ fn orelseCatchExpr(
 
     _ = try addZIRInst(mod, &then_scope.base, src, zir.Inst.Break, .{
         .block = block,
-        .operand = try expr(mod, then_sub_scope, branch_rl, rhs),
+        .operand = try rlWrap(mod, then_sub_scope, .{ .ref = {} }, try expr(mod, then_sub_scope, branch_rl, rhs)),
```

Comment:
This part doesn't look right - what are you trying to do here?

Reply:
It expected a pointer, but found a value, so making a pointer. Is `addZIRUnOp(mod,then_sub_scope,.ref, ...)` more idomatic? I thought a rlWrap was the best.

Reply:
What do you mean "it expected a pointer"?
The docs say:

```
        /// Return a value from a `Block`.
        @"break",
```

and the implementation has nothing there to suggest the value should be a pointer.